code/game/machinery/atmo_control.dm

```diff
@@ -545,8 +546,13 @@
 
 	add_fingerprint(usr)
 
+	if(href_list["in_set_rate"])
+		var/response = input(usr, "Set new flow rate, in L/s. \[0-[MAX_TRANSFER_RATE]\]") as num
+		volume_rate_setting = text2num(response)
+		volume_rate_setting = between(0, volume_rate_setting, MAX_TRANSFER_RATE)
```

Comment:
Can use Clamp here instead. Clamp is a define instead of a proc so less overhead. clamp is also be integrated in byond 513 so once 513 is stable we can replace all `Clamp` calls with `clamp`